extensions/PhabBugz/lib/Revision.pm

```diff
@@ -389,4 +404,20 @@ sub set_policy {
     $self->{set_policy}->{$name} = $policy;
 }
 
-1;
\ No newline at end of file
+sub add_project {
+    my ( $self, $project ) = @_;
+    $self->{add_projects} ||= [];
+    my $project_phid = blessed $project ? $project->phid : $project;
+    return undef unless $project_phid;
+    push @{ $self->{add_projects} }, $project_phid;
+}
+
+sub remove_project {
+    my ( $self, $project ) = @_;
+    $self->{remove_projects} ||= [];
+    my $project_phid = blessed $project ? $project->phid : $project;
+    return undef unless $project_phid;
+    push @{ $self->{remove_projects} }, $project_phid;
+}
+
+1;
```

Comment:
You were asking in IRC before, the 1; at the end is because Perl modules need to always return a true value when a calling file loads (require or use) the module.

Reply:
If we had a new enough version of perl, like one from 5 years ago, I think package block syntax returns a true value.

```perl
package Foo::Bar {


}
```
some day...